app/views/profile/customer_cabinet.html.erb

```diff
@@ -41,3 +41,7 @@
       <% end %>
     </div>
 </section>
+
+<section>
+  <%= flash[:notice] = "Один, или несколько ваших исполнителей сообщили о выполнении проектов. Необходимо подтвердить выполнение." if @implementation_project %>
```

Comment:
and here? localization

Reply:
Considering that this place will be redone at least once for the external interface, I skipped this part for faster creation and discussion of this function. And the rest, I’m only for